packages/server/jiraImagesHandler.ts

```diff
@@ -8,13 +7,14 @@ import uWSAsyncHandler from './graphql/uWSAsyncHandler'
 import getRedis from './utils/getRedis'
 
 const getImageFromCache = async (fileName: string, tryAgain: boolean) => {
```

Comment:
+1 should we rename `fileName` to `imageUrlHash` to be consistent?

Reply:
done